src/next-collection-view.js

```diff
@@ -280,14 +287,19 @@ const CollectionView = Backbone.View.extend({
       this._addChildModels(this.collection.models);
     }
 
-    this._showChildren();
+    this._render();
 
     this._isRendered = true;
 
     this.triggerMethod('render', this);
     return this;
   },
 
+  // For overriding in CompositeView
```

Comment:
This was added so that composite view can define what occurs during the render.. it was actually important to do it this way so that children could be destroyed while still attached.  This clears up one of the caveats